action.yml

```diff
@@ -7,11 +7,9 @@ inputs:
   package:
     description: 'The package whose API to check for semver (in Package Id Specification format, see https://doc.rust-lang.org/cargo/reference/pkgid-spec.html for reference). If not set, all packages in the workspace are processed.'
     required: false
-    default: ''
   manifest-path:
     description: 'Path to Cargo.toml of crate or workspace to check.'
```

Comment:
The inputs here are generally great at documenting what the default behavior is when a `required: false` input isn't set. Let's document that behavior here as well.

Reply:
I've done this and the above one in https://github.com/obi1kenobi/cargo-semver-checks-action/pull/25